aur/scenefx/PKGBUILD

```diff
@@ -1,9 +1,10 @@
 # Maintainer: Erik Reider <erik.reider@protonmail.com>
 
 _pkgname=scenefx
-_pkgver=0.2
+_pkgver_micro=".1"
+_pkgver="0.2"
```

Comment:
q: is this typical for AUR packages? I think we just have pkgver include the minor ver in prior swayfx releases

Reply:
not a blocker just curious

Reply:
Smort :)